### ISSUE TITLE
incorrect date time in log

### ISSUE THREAD

**Issue by zhangzq** [odygrd/quill#127]
On a server (with ubuntu 20.04), the quill log shown following datetime 

```
1971-01-30 23:54:07.685312740
```

The datetime is changed correctly with realtime, I think only the basetime is set uncorrectly.

I have checked `date -R`, and the timestamp of system log, python logging, all correct.

Anything I can do?

**Comment by anluoma** [comment]
Hi,

We had a little similar strange issue on Windows server last week as well. We checked that the server system time was updated 21-8-26. Later on logs showed old dates and hour part of the time was somehow skewed:

for example: 
`
2021-08-29 106040410.436 +0300 [6620] LOG_WARNING
2021-08-30 82:07:15.436 +0300 [2352] LOG_WARNING
`
Dunno if its same issue but something is not right.

**Comment by odygrd** [comment]
I am going to try to investigate further. if anyone has seen this issue again or can provide more info please let me know

**Comment by odygrd** [comment]
@zhangzq when you say ‘ The datetime is changed correctly with realtime, I think only the basetime is set uncorrectly.’ 

Do you mean that the time appeared correctly and only the dates were wrong?

**Comment by odygrd** [comment]
@anluoma can you give me the format pattern for the timestamp you are using please in `set_pattern(...)` ?

**Comment by odygrd** [comment]
Thanks for reporting. I managed to reproduce the issue by changing the system's clock while the logger was running. This should be fixed in 6ac18e67e98da5926d9455d351c961e343e68fb2

Please open a new ticket if you see any similar issue again

**Comment by anluoma** [comment]
Hi,

really nice to hear this was fixed. Sorry that I didn't spot this earlier. We use following pattern
```
QUILL_STRING("%(ascii_time) [%(thread)] LOG_%(level_name) %(logger_name) - %(message)"),
      "%Y-%m-%d %H:%M:%S.%Qms %z"
```
